smallrye-reactive-messaging-jms/src/test/java/io/smallrye/reactive/messaging/jms/JmsConnectorTest.java

```diff
@@ -31,7 +25,7 @@ public void testWithString() {
         map.put("mp.messaging.incoming.jms.destination", "queue-one");
         MapBasedConfig config = new MapBasedConfig(map);
         addConfig(config);
-        WeldContainer container = deploy(PayloadConsumerBean.class, ProducerBean.class);
+        WeldContainer container = deploy(PayloadConsumerBean.class, ProducerBean.class, TestJson.class);
```

Comment:
Instead of passing TestJson every time, can you add it to the deploy method directly?

Reply:
Sure if you don't mind.

Reply:
What about `#initWithoutConnectionFactory`? I think it suits better?

Reply:
Yes!